controller/workitemtype_blackbox_test.go

```diff
@@ -220,6 +248,10 @@ func CreateWorkItemType(id uuid.UUID, spaceID uuid.UUID) app.CreateWorkitemtypeP
 		Request: &http.Request{Host: "api.service.domain.org"},
 	}
 	spaceSelfURL := rest.AbsoluteURL(reqLong, app.SpaceHref(spaceID.String()))
+
+	// fetch category
+	category := getCategory(db)
```

Comment:
i have removed this function and instead i use category ids hard-coded in the repository.